docs_user/modules/proc_adopting-the-compute-service.adoc

```diff
@@ -2,44 +2,29 @@
 
 = Adopting the {compute_service}
 
-[NOTE]
-This example scenario describes a simple single-cell setup. Real
-multi-stack topology recommended for production use results in different
-cells DBs layout, and should be using different naming schemes (not covered
-here this time).
+To adopt the {compute_service_first_ref}, you patch an existing `OpenStackControlPlane` custom resource (CR) where the {compute_service} is disabled. The patch starts the service with the configuration parameters that are provided by the {rhos_prev_long} ({OpenStackShort}) environment. The following procedure describes a single-cell setup.
+
+//[NOTE]
+//The following example scenario describes a single-cell setup. Real
+//multi-stack topology that is recommended for production use results in cells having a different database layout, and should use different naming schemes. kgilliga: We might reinstate this note after multi-cell is finished in Feature Release 1.
 
 .Prerequisites
 
-* Previous Adoption steps completed. Notably,
- ** the xref:migrating-databases-to-mariadb-instances_migrating-databases[Migrating databases to MariaDB instances]
-must already be imported into the control plane MariaDB;
- ** the xref:adopting-the-identity-service_adopt-control-plane[Adopting the Identity service] needs to be imported;
- ** the xref:adopting-the-key-manager-service_adopt-control-plane[Adopting the Key Manager service] needs to be imported;
- ** the xref:adopting-the-placement-service_{context}[Adopting the Placement service] needs to be imported;
- ** the xref:adopting-the-image-service_adopt-control-plane[Adopting the Image service] needs to be imported;
- ** the xref:migrating-ovn-data_migrating-databases[Migrating OVN data] need to be imported;
- ** the xref:adopting-the-networking-service_adopt-control-plane[Adopting the Networking service] needs to be imported;
-** the {bare_metal} needs to be imported;
-//kgilliga:I removed the link because it did not work. I might rewrite this section anyway.
- ** Required topology-specific service configuration. For more information, see
-xref:proc_retrieving-topology-specific-service-configuration_migrating-databases[Retrieving topology-specific service configuration].
- ** {rhos_prev_long} services have been stopped. For more information, see xref:stopping-openstack-services_migrating-databases[Stopping {rhos_prev_long} services].
-* Define the following shell variables. The values that are used are examples. Replace these example values with values that are correct for your environment:
+* Define the following shell variables. Replace the following example values with the values that are correct for your environment:
```

Comment:
As this is a prerequisite, the user should already have performed it. So reword to: 

```suggestion
* You have defined the following shell variables. Replace the following example values with the values that are correct for your environment:
```